packages/frontend/core/src/modules/peek-view/view/modal-container.tsx

```diff
@@ -128,6 +128,17 @@ export const PeekViewModalContainer = forwardRef<
         const targets = contentClip;
         const lockSizeEl = content;
 
+        // if target has no bounding client rect, find its parent that has bounding client rect
+        while (target && !target.getBoundingClientRect().width) {
```

Comment:
no stop condition or max loop control? looks a bit ka

Reply:
will give a max iteration of 10